multimedia/ffmpeg6/Portfile

```
@@ -539,6 +544,20 @@ This build of ${name} includes no GPLed or nonfree code and is therefore license
 "
 }
 
+notes-append "
+--------
+To use the ${name} command-line programs without suffix, add
+${prefix}/libexec/ffmpeg6/bin to your \$PATH,
```

Please avoid hard newlines so that MacPorts can add newlines where needed based on the user's terminal width.
```suggestion
To use the ${name} command-line programs without suffix, add\
${prefix}/libexec/ffmpeg6/bin to your \$PATH,\
```